app/views/unattended/provisioning_templates/registration/linux_registration_default.erb

```diff
@@ -0,0 +1,67 @@
+<%#
+kind: registration
+name: Linux registration default
+model: ProvisioningTemplate
+oses:
+- CentOS
+- Fedora
+-%>
+#!/bin/bash
+
+set -e
+
+# This script enrolls a host with Foreman.
+# Host: <%= @host.name %>
+<% if @host.puppetmaster.present? -%>
+
+# Setup puppet repositories
+
+rpm -Uvh https://yum.puppetlabs.com/puppet6/puppet6-release-el-7.noarch.rpm
```

Comment:
Don't we have a snippet for this? I also think this should be a bit more flexible since this is very much EL7 only.